docker/Dockerfile

```diff
@@ -20,7 +20,10 @@ RUN curl https://get.docker.com/builds/Linux/x86_64/docker-1.12.3.tgz \
 RUN pip install toil==3.3.5
 
 # Install toil-rnaseq
-RUN pip install toil-rnaseq==3.2.1
+COPY toil-rnaseq.tar.gz .
+RUN pip install toil-rnaseq.tar.gz
+RUN rm toil-rnaseq.tar.gz
```

Comment:
If you want to `rm` the .tar.gz you should put it on the same line as the `pip install` command with `&&` . Because of the way Docker builds layers the current implementation actually increases the image size.